generator/templates/Default/src/plugins/vuetify/FontawesomePro.js

```diff
@@ -0,0 +1,262 @@
+  faPencilAlt,
+  faStar,
+  faSync,
+  faStepBackward,
+  faStepForward,
+  faArrowsAltV,
+  faPaperclip,
+  faPlus,
+  faMinus,
+  faUser,
+  faCog,
+  faHome,
+  faHandshake,
+  faBusinessTime,
+  faUsers,
+  faBriefcase,
+  faChartBar,
+  faUpload,
+  faFile,
+  faFileVideo,
+  faFileWord,
+  faFileExcel,
+  faFilePowerpoint,
+  faFilePdf,
+  faTriangle,
+  faHexagon,
+  faBell,
+  faSearch,
+  faCar,
+  faTrain,
+  faPhone,
+  faEnvelope,
+  faFax,
+  faGlobeEurope,
+  faMapMarkerAlt,
+  faCalendarAlt,
+  faClock,
+  faList,
+  faBirthdayCake,
+  faHourglassHalf,
+  faQuestionCircle,
+  faEllipsisV,
+  faEye,
+  faEyeSlash,
+  faFolder,
+  faFolderOpen,
+  faGenderless,
+  faMars,
+  faVenus,
+  faFilter,
+  faUserUnlock,
+  faBuilding,
+  faLockAlt,
+  faUnlockAlt,
+);
+
+const icons = {
+  /*vuetify default icons*/
+  complete: ['far', 'check'],
+  cancel: ['fal', 'times'],
+  close: ['fal', 'times'],
+  delete: ['fal', 'times'],
+  clear: ['fal', 'times'],
+  success: ['far', 'check-circle'],
+  info: ['far', 'info-circle'],
+  warning: ['far', 'exclamation'],
+  error: ['far', 'exclamation-triangle'],
+  prev: ['far', 'chevron-left'],
+  next: ['far', 'chevron-right'],
+  checkboxOn: ['fas', 'check-square'],
+  checkboxOff: ['far', 'square'],
+  checkboxIndeterminate: ['fas', 'minus-square'],
+  delimiter: ['fas', 'circle'],
+  sort: ['far', 'sort-up'],
+  expand: ['far', 'chevron-down'],
+  menu: ['far', 'bars'],
+  subgroup: ['far', 'caret-down'],
+  dropdown: ['far', 'caret-down'],
+  radioOn: ['far', 'dot-circle'],
+  radioOff: ['far', 'circle'],
+  edit: ['far', 'pencil-alt'],
+  ratingEmpty: ['far', 'star'],
+  ratingFull: ['fas', 'star'],
+  ratingHalf: ['fas', 'star-half'],
+  loading: ['far', 'sync'],
+  first: ['far', 'step-backward'],
+  last: ['far', 'step-forward'],
+  unfold: ['far', 'arrows-alt-v'],
+  file: ['far', 'paperclip'],
+  plus: ['far', 'plus'],
+  minus: ['far', 'minus'],
+  /*custom icons*/
+
+  profile: ['far', 'user'],
+  settings: ['far', 'cog'],
+  home: ['far', 'home'],
+  notifications: ['far', 'bell'],
+  search: ['far', 'search'],
+  birthday: ['far', 'birthday-cake'],
+  invited: ['far', 'hourglass-half'],
+  unknown: ['far', 'question-circle'],
+  fabDots: ['fas', 'ellipsis-v'],
+  visible: ['far', 'eye'],
+  invisible: ['far', 'eye-slash'],
+  filter: ['far', 'filter'],
+
+  vacancy: ['far', 'handshake'],
+  shift: ['far', 'business-time'],
+  candidate: ['far', 'users'],
+  employer: ['far', 'briefcase'],
+  report: ['far', 'chart-bar'],
+  role: ['far', 'user-unlock'],
+  location: ['far', 'building'],
+  department: ['far', 'building'],
+  position: ['far', 'briefcase'],
+
+  upload: ['far', 'upload'],
+  fileUnknown: ['far', 'file'],
+  fileVideo: ['far', 'file-video'],
+  fileWord: ['far', 'file-word'],
+  fileExcel: ['far', 'file-excel'],
+  filePowerpoint: ['far', 'file-powerpoint'],
+  filePdf: ['far', 'file-pdf'],
+  folderOpen: ['far', 'folder-open'],
+  folderClosed: ['far', 'folder'],
```

Comment:
Is this global? I'm not sure but i don't think so.

Reply:
Not sure what you mean: these are used in the scaffold. And yes, it is available as <v-icon>$****</v-icon>

Reply:
The same as the other comments, except for this one i'm not sure.
So are these icons supposed to be global/in the scaffold?
I'm not sure but i don't think so.

Reply:
```
  fileUnknown: ['far', 'file'],
  fileVideo: ['far', 'file-video'],
  fileWord: ['far', 'file-word'],
  fileExcel: ['far', 'file-excel'],
  filePowerpoint: ['far', 'file-powerpoint'],
  filePdf: ['far', 'file-pdf'],
```

Excel and powerpoint could be dropped, these are not used. The others are actually used in KFileField. Removing those will result in a broken component.